CHANGELOG.md

```
@@ -9,37 +9,29 @@
 
 ## [1.6.1](https://github.com/VilnaCRM-Org/php-service-template/compare/v1.6.0...v1.6.1) (2025-05-05)
 
-
 ### Bug Fixes
 
-* dependabot.yml ([be0056b](https://github.com/VilnaCRM-Org/php-service-template/commit/be0056bf3f16bb79826d51b6aadf42aa3d729437))
-
-
+- dependabot.yml ([be0056b](https://github.com/VilnaCRM-Org/php-service-template/commit/be0056bf3f16bb79826d51b6aadf42aa3d729437))
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)